akka-stream/src/main/scala/akka/stream/javadsl/Sink.scala

```diff
@@ -99,6 +91,33 @@ object Sink {
     new Sink(scaladsl.Sink.head[In])
 
   /**
+   * Sends the elements of the stream to the given `ActorRef`.
+   * If the target actor terminates the stream will be cancelled.
+   * When the stream is completed successfully the given `onCompleteMessage`
+   * will be sent to the destination actor.
+   * When the stream is completed with failure a [[akka.actor.Status.Failure]]
+   * message will be sent to the destination actor.
+   *
+   * It will request at most `maxInputBufferSize` number of elements from
+   * upstream, but there is no back-pressure signal from the destination actor,
+   * i.e. if the actor is not consuming the messages fast enough the mailbox
+   * of the actor will grow. For potentially slow consumer actors it is recommended
+   * to use a bounded mailbox with zero `mailbox-push-timeout-time` or use a rate
```

Comment:
yes, but the target Actor can still watch this one and thereby know when the stream ended